PiwikPlugin.inc.php

```diff
@@ -103,8 +103,8 @@ function manage($args, $request) {
 	 * Override the builtin to get the correct template path.
 	 * @return string
 	 */
-	function getTemplatePath() {
-		return $this->getTemplateResourceName() . ':templates/';
+	function getTemplatePath($inCore = false) {
+		return $this->getTemplateResource() . ':templates/';
```

Comment:
For the `master` branch, the `getTemplatePath` function should be removed entirely. Anything that needs to fetch a template name it using `Plugin::getTemplateResource`, e.g.:

    $this->getTemplateResource('display.tpl')

Reply:
Done @asmecher.